external/pom.xml

```diff
@@ -71,6 +71,11 @@
 			<groupId>org.opencds.cqf.cql</groupId>
 			<artifactId>evaluator.fhir</artifactId>
 		</dependency>
+		<!-- <dependency>
+			<groupId>com.converter</groupId>
+			<artifactId>cqf-ruler-v1-converter-plugin</artifactId>
+			<version>1.0.0</version>
+		</dependency> -->
```

Comment:
without adding the plugin as a dependency to the server I can't import it to Application.java, but if I import it then I get a circular dependency error